apps/server/src/modules/meta-tag-extractor/service/url-handler/board-url-handler.ts

```diff
@@ -7,7 +7,7 @@ import { AbstractUrlHandler } from './abstract-url-handler';
 
 @Injectable()
 export class BoardUrlHandler extends AbstractUrlHandler implements UrlHandler {
-	patterns: RegExp[] = [/\/course-rooms\/(.*?)\/board\/?$/i];
+	patterns: RegExp[] = [/\/boards\/([0-9a-z]+)$/i];
```

Comment:
If fancy.
`	patterns: RegExp[] = [/\/boards\/([0-9a-z]{24})$/i];
`

But also okay like it is.

Reply:
I see your idea, but I would like to leave it like that to be less precise in case we change in the future, but thx :)

Reply:
Well, on the other side if we'd be more precise we could prevent clashes with other paths right? Either way it works but i would also prefer the "real" mongoid regexp.